### PR TITLE
Add basic embedded isReady support

### PR DESCRIPTION
**Pull request by crow**
Adds support for checking status of embedded views

### CODE REVIEW

File: ios/AirshipFrameworkProxy/AirshipDelegate.swift [hunk 1]
```diff
@@ -107,6 +125,16 @@ class AirshipDelegate: NSObject,
 
     }
 
+    func embeddedInfoUpdated(embeddedInfo: [AirshipEmbeddedInfo]) {
```
Comment (rlepinski) [inline ios/AirshipFrameworkProxy/AirshipDelegate.swift:128]: Do you want to add a function to Automation that is `getPendingEmbedddedInfos(embeddedId: String?)`

File: ios/AirshipFrameworkProxy/AirshipProxyEvent.swift [hunk 2]
```diff
@@ -163,4 +165,15 @@ struct AuthorizedNotificationSettingsChangedEvent: AirshipProxyEvent {
 
 }
 
+struct EmbeddedInfoUpdatedEvent: AirshipProxyEvent {
+    let type: AirshipProxyEventType = .embeddedInfoUpdated
+    let body: [String: Any]
+
+    init(embeddedInfo: [AirshipEmbeddedInfo]) {
+        self.body = [
+            "embeddedIds": embeddedInfo.map{ $0.embeddedID }
```
Comment (rlepinski) [inline ios/AirshipFrameworkProxy/AirshipProxyEvent.swift:174]: pending: [EmbeddedInfo]

Comment (rlepinski) [reply ios/AirshipFrameworkProxy/AirshipProxyEvent.swift:174]: EmbeddedInfo could just have an id for now

File: ios/AirshipFrameworkProxy/AirshipDelegate.swift (hunk 1 above)
Comment (crow) [reply ios/AirshipFrameworkProxy/AirshipDelegate.swift:128]: sure

**Pull request merged**